composer.json

```diff
@@ -12,8 +12,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
-        "guzzle/guzzle": "~3.5"
+        "php": "~7.1",
+        "guzzlehttp/guzzle": "^6.3.3",
+        "symfony/event-dispatcher": "^4.1"
```

Comment:
Do we actually need to depend on event-dispatcher? 

```
$ grep -rih ^use ./{src,test} | sort -u
use GuzzleHttp\Client;
use GuzzleHttp\HandlerStack;
use GuzzleHttp\Handler\MockHandler;
use GuzzleHttp\Psr7\Response as HttpResponse;
use Toggl\Api\Response as ApiResponse;
use Toggl\Api\TogglApiClientV8;
use Toggl\Common\TogglClient;
use Toggl\Common\TogglClientAuthPlugin;
use Toggl\Reports\TogglReportsApiClientV2;
use Toggl\Test\Api\Response\MeTest;
use Toggl\Test\Api\Response\ProjectTest;
use Toggl\Test\Api\Response\ProjectsTest;
use Toggl\Test\Api\Response\WorkspaceUsersTest;
use Toggl\Test\Api\Response\WorkspacesTest;
```

Reply:
Apparently not.

```
[slerner:~/Sites/toggl-php] composer why symfony/event-dispatcher
webbj74/toggl-php  dev-ENT-3197  requires  symfony/event-dispatcher (^4.1)
```